tests/kubeTest.sh

```diff
@@ -47,6 +47,7 @@ done
 [[ -z ${NAMESPACE} ]] && NAMESPACE="$(generate_namespace)"
 
 if [[ -z ${ISTIOCLI} ]]; then
+    echo "Downloading istioctl from ${ISTIOCTL_URL}/istioctl-linux"
     wget -q -O "${TEST_DIR}/istioctl" "${ISTIOCTL_URL}/istioctl-linux" || error_exit "Could not download istioctl"
```

Comment:
well if the build passes, nobody looks at the logs so a few more lines don't matter, and if it fails; it's useful to know exactly what it used (in this case it wasn't obvious to me and I think it will help next time too)